1-exercises/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,8 @@
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+let newArray = arr.forEach(value => console.log(value % 3 === 0 ? "Fizz" : value % 5 === 0 ? "Buzz" : value % 3 === 0 && value % 5 === 0 ? "FizzBuzz" : value));
```

Comment:
This isn't giving the correct result for 15, can you see what the issue might be? there is a simple solution to the problem, see if you can spot it